spoc/snipping/__init__.py

```diff
@@ -1,2 +1,2 @@
 """Snipping module of spoc that provides Snipper class and snipping strategies"""
-from spoc.snipping.snipper import Snipper
\ No newline at end of file
+from spoc.snipping.snipper import Snipper
```

Comment:
I've never used Snipper as a fast production of a list of pileups. The main problem with `Snipper.snip` output is that it produces only pileups without an annotation of them, e.g., offsets. The desired output would be a dictionary with keys as parameters of individual strategies and values as pileups.

Reply:
Good point! I would say, however, that this is not the scope of this PR. We can talk about the return value of `Snipper.snip` when the actual snipping with dynamic contact orders is implemented in https://trello.com/c/kxwfESdV/60-cct-triplets-anchoredwindowstrategy-with-cc-as-anchor

Reply:
I thought a bit about what I would like to have in the Snipper class and what would be useful to have there in general, here's a ticket: https://trello.com/c/Nn0Uu0Al.